source/enumerate-collections.rst

```diff
@@ -303,6 +314,8 @@ Example return (a cursor which returns documents, not a simple array)::
 When returning this information as a cursor, a driver SHOULD use the
 method name ``listCollections`` or an idiomatic variant.
 
+Drivers SHOULD specify the nameOnly option when executing the ``listCollections`` command for this method.
```

Comment:
I don't think this method should specify nameOnly since it's about getting the full collection information. Perhaps drivers SHOULD/MAY allow the nameOnly option to be passed?